src/components/emailCapture/emailCaptureForm/index.js

```diff
@@ -12,7 +12,6 @@ class BaseEmailCaptureForm extends React.Component {
         <FlexCol mobile={{width: 4}} desktop={{width: 6, span: 3}}>
           <form onSubmit={onEmailSubmit}>
             <Input
-              style={{ width: '100%', boxSizing: 'border-box' }}
```

Comment:
Inputs are now 100% width by default and all elements have `box-sizing: border-box` by default